roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -96,6 +96,24 @@ kubernetesMasterConfig:
   - v1beta3
   - v1
 {% endif %}
+{% if openshift.master.builddefaults_http_proxy or openshift.master.builddefaults_https_proxy
+   and openshift.common.version_gte_3_2_or_1_2 %}
+  admissionConfig:
+    pluginConfig:
+      BuildDefaults:
+        configuration:
+          apiVersion: v1
+          kind: BuildDefaultsConfig
+          gitHTTPProxy: {{ openshift.master.builddefaults_git_http_proxy }}
+          gitHTTPSproxy: {{ openshift.master.builddefaults_git_https_proxy }}
+          env:
+          - name: HTTP_PROXY
+            value: '{{ openshift.master.builddefaults_http_proxy }}'
+          - name: HTTPS_PROXY
+            value: '{{ openshift.master.builddefaults_https_proxy }}'
+          - name: NO_PROXY
+            value: '{{ openshift.master.builddefaults_no_proxy }}'
```

Comment:
@csrwng Can you comment on this? what we should do here? Mainly we're trying to come up with what the default NO_PROXY should be for the builddefaults controller. What endpoints, internal to the cluster, are used during the build phase? I'm wondering if there aren't any since the only thing that requires internal access would be the docker push and that's happening outside of the build environment via the docker daemon on the node.